pkg/provider/operators.go

```diff
@@ -33,12 +38,32 @@ type Operator struct {
 	Org              string                            `yaml:"org" json:"org"`
 	Version          string                            `yaml:"version" json:"version"`
 	Channel          string                            `yaml:"channel" json:"channel"`
+	PreflightResults plibRuntime.Results
 }
 
 func (op *Operator) String() string {
 	return fmt.Sprintf("csv: %s ns:%s subscription:%s", op.Name, op.Namespace, op.SubscriptionName)
 }
 
+func (op *Operator) SetPreflightResults() error {
+	bundleImage := op.InstallPlans[0].BundleImage
+	indexImage := op.InstallPlans[0].IndexImage
+	check := plib.NewOperatorCheck(bundleImage, "/Users/bpalm/.kube/config", indexImage)
```

Comment:
We should add a way to retrieve the kubeconfig file used by the clientsholder so you can use it here.

Reply:
I'm currently open to suggestions about how to gather that info.  😄 I wasn't able to find anything straight forward about how to retrieve a KUBECONFIG from the client.

Reply:
Feature added in #644. 🎉